x-pack/plugins/observability/public/pages/slos/components/grouped_slos/group_list_view.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useState } from 'react';
+import { ALL_VALUE } from '@kbn/slo-schema';
+import { EuiPanel, EuiAccordion, EuiTablePagination } from '@elastic/eui';
+import { useFetchSloList } from '../../../../hooks/slo/use_fetch_slo_list';
+import { useFetchActiveAlerts } from '../../../../hooks/slo/use_fetch_active_alerts';
+import { useFetchRulesForSlo } from '../../../../hooks/slo/use_fetch_rules_for_slo';
+
+import { SloListItems } from '../slo_list_items';
+
+interface Props {
+  isCompact: boolean;
+  group: string;
+  kqlQuery: string;
+}
+
+export function GroupListView({ isCompact, group, kqlQuery }: Props) {
+  const query = kqlQuery ? `"slo.tags": ${group} and ${kqlQuery}` : `"slo.tags": ${group}`;
+  const [page, setPage] = useState(0);
+  const ITEMS_PER_PAGE = 10;
+  // TODO get sortBy and sortDirection from parent
+  const {
+    isLoading,
+    isRefetching,
+    isError,
+    data: sloList,
+  } = useFetchSloList({
+    kqlQuery: query,
+    perPage: ITEMS_PER_PAGE,
+    page: page + 1,
+  });
+  const { results = [], total = 0 } = sloList ?? {};
+
+  const sloIdsAndInstanceIds = results.map(
+    (slo) => [slo.id, slo.instanceId ?? ALL_VALUE] as [string, string]
+  );
+  const { data: activeAlertsBySlo } = useFetchActiveAlerts({ sloIdsAndInstanceIds });
+  const { data: rulesBySlo } = useFetchRulesForSlo({
+    sloIds: sloIdsAndInstanceIds.map((item) => item[0]),
+  });
+
+  const handlePageClick = (pageNumber: number) => {
+    setPage(pageNumber);
+    // storeState({ page: pageNumber });
+  };
+
+  return (
+    <EuiPanel>
+      <MemoEuiAccordion buttonContent={group} id={group} initialIsOpen={false}>
+        <>
+          <SloListItems
+            activeAlertsBySlo={activeAlertsBySlo}
+            sloList={results}
+            loading={isLoading || isRefetching}
+            error={isError}
+            isCompact={isCompact}
+            rulesBySlo={rulesBySlo}
+          />
+          <EuiTablePagination
+            pageCount={Math.ceil(total / ITEMS_PER_PAGE)}
+            activePage={page}
+            onChangePage={handlePageClick}
+            itemsPerPage={ITEMS_PER_PAGE}
+          />
```

Comment:
Couldn't you reuse <SlosView /> component directly?
```
<SlosView
          sloList={results}
          loading={isLoading || isRefetching}
          error={isError}
          isCompact={isCompact}
          sloView={view}
        />
```

Reply:
Yep good idea! I was actually refactoring this as part of [Group by and view modes ](https://github.com/elastic/kibana/issues/174332) and your suggestion works like a charm. I pushed the change

https://github.com/elastic/kibana/assets/2852703/55eef08d-5c13-41f2-9c73-16af9d9f37c6